include/ofi_hmem.h

```diff
@@ -172,6 +172,12 @@ void cuda_gdrcopy_to_dev(uint64_t handle, void *dev,
 			 const void *host, size_t size);
 void cuda_gdrcopy_from_dev(uint64_t handle, void *host,
 			   const void *dev, size_t size);
+ssize_t cuda_gdrcopy_to_dev_iov(uint64_t handle, const struct iovec *iov,
+                                size_t iov_count, uint64_t iov_offset,
+                                const void *hostptr, size_t len);
```

Comment:
nit: Other functions nearby just use 'host' as the parameter name.  'ptr' is implied by it being a void *.  Similar comment applies below.

Reply:
Good point. Addressed.